lib/OBOClient/index.js

```diff
@@ -52,7 +53,7 @@ function send (userToken, conversationId, message, data) {
     const options = {
       hostname: hostname,
       port: SymConfigLoader.SymConfig.agentPort,
-      path: `/agent/v4/stream/${conversationId}/message/create`,
+      path: PathBuilder.buildAgentPath(`/agent/v4/stream/${conversationId}/message/create`),
```

Comment:
I'm not sure about this line because in Nodejs if you use `` , ${conversationId} will be understand as a param. But if you use ' ', ${conversationId} will be understand as a string '${conversationId}'.

Reply:
ah yes, it should work because conversationId will be replaced to the string passed to the function

Reply:
We want the backticks `` so that ${conversationId} is replaced by the actual variable value, see here for more information: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals